manifests/d/DygmaLabs/Bazecor/1.7.0/DygmaLabs.Bazecor.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created with WinGet Updater using komac v2.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: DygmaLabs.Bazecor
+PackageVersion: 1.7.0
+InstallerType: exe
+Scope: user
+ReleaseDate: 2025-03-27
+Installers:
+- Architecture: x86

```

```suggestion
- Architecture: x64
```

Maybe the architecture is the issue. i don't have any x86 VMs but it installs fine on x64.

---

Possibly, lets try